packages/configuration-validation/test/lib.test.js

```diff
@@ -70,14 +76,76 @@ describe('Configuration Validation', () => {
 
       expect(buildConfigObject(passedConfig)).toEqual({
         clientId: '{clientId}',
+        client_id: '{client_id}',
         issuer: '{issuer}',
         redirectUri: '{redirectUri}',
+        redirect_uri: '{redirect_uri}',
+        storage: '{storage}',
+        autoRenew: '{autoRenew}',
+        auto_renew: '{auto_renew}',
```

Comment:
Do we use `auto_renew` somewhere?  If it's deprecated, should we remove it from the test I/O in all but a "deprecated still works" test?

Reply:
It has been in the documentation for awhile, especially on the react side, where it was actually required to use the underscore syntax until very recently. I think we will want to let it settle for awhile before removing

Reply:
Makes sense - perhaps a comment here to say `// from okra-react vx.x ` or whenever, so we know when it can be removed?